xml/System.Net.Http/HttpVersionPolicy.xml

```diff
@@ -54,7 +62,17 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Tries to uses highest available version, downgrading only to the requested version, not bellow. Throwing <see cref="HttpRequestException" /> if a connection with higher or equal version cannot be established.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+If the server supports higher than requested version, either negotiated via ALPN (H2) or advertised via Alt-Svc (H3), as well as secure connection is being requested, the result is the highest available one. Otherwise, downgrades to the <see cref="HttpRequestMessage.Version" />.
```

Comment:
```suggestion
If the server supports a higher version than the requested version (either negotiated via ALPN (H2) or advertised via Alt-Svc (H3)) and a secure connection is requested, the result is the highest available version. Otherwise, the version downgrades to <see cref="HttpRequestMessage.Version" />.
```